lib/Raven/Configuration.php

```diff
@@ -0,0 +1,446 @@
+<?php
+
+/*
+ * This file is part of Raven.
+ *
+ * (c) Sentry Team
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Raven;
+
+use Composer\CaBundle\CaBundle;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Configuration container for the Sentry client.
+ *
+ * @author Stefano Arlandini <sarlandini@alice.it>
+ */
+class Configuration
```

Comment:
I am a fan of the configuration class FWIW. This is a nice thinga bout the Ruby SDK that I wish we would refactor in Python. It just makes it more obvious where config lives, and can then easily be exposed from the base client (vs having to know which attributes are configuration or state)